v2/pyext/src/analysis.py

```diff
@@ -700,16 +700,16 @@ def split_into_two_POFs(self):
         import random
         # Split list of output files into two sets
 
-        n_output_files = range(len(self.output_files))
+        n_output_files = list(range(len(self.output_files)))
         random.shuffle(n_output_files)
 
         pof1 = ParseOutputFile(self.output_files[n_output_files[0]])
-        for i in n_output_files[1:len(n_output_files)/2]:
+        for i in n_output_files[1:int(len(n_output_files)/2)]:
```

Comment:
The usual way to handle this in Python is to use the `//` operator, i.e.

             for i in n_output_files[1:len(n_output_files) // 2]:

You'll want to add `from __future__ import division` at the top of the file so that this works in Python 2 too.